archetypes/data-playground.md

```diff
@@ -12,12 +12,13 @@ producer_link: "https://finnhub.io"
 data: "Add Details About the Data Provided Here"
 is_account_required: false
 license: "Add Data Source License Type Here (Ex. Free, Commercial)"
+license_warning: "Add License description if applicable"
 is_api_key_required: false
 api_type: "Add API Type(s) Provided Here and Separate Multiple Types With A Comma"
 sdks: "Add Available SDKs Here and Separate Multiple SDKs With A Comma"
 limits: "Add API Limit Information Here, If Available"
 
 ---
 
-<!-- Add content for data playground here, including a table for data products if available -->
+Add content for data playground here, including a table for data products if available
```

Comment:
I think that this should remain commented out. Please look at the reply about why this was done here: https://github.com/rotationalio/rotational.io/pull/166#discussion_r1223490948